distsql/select_result.go

```diff
@@ -167,6 +172,22 @@ func (r *selectResult) getSelectResp() error {
 	}
 }
 
+func (r *selectResult) updateCopRuntimeStats(callee string) {
+	if r.ctx.GetSessionVars().StmtCtx.RuntimeStatsColl == nil ||
+		callee == "" || len(r.selectResp.GetExecutionSummaries()) != len(r.copPlanIDs) {
```

Comment:
if `len(r.selectResp.GetExecutionSummaries()) != len(r.copPlanIDs)`, we should log an error

Reply:
fixed.